app/tests/test_collections_summaries.py

```diff
@@ -173,3 +173,12 @@ def test_update_collection_summaries_asset_update(self):
             "Collection's summaries[proj:epsg] has not been correctly "
             "updated after asset has been inserted."
         )
+
+    def test_update_collection_summaries_none_variant(self):
+        # update a variant, that as been None as a start value
+        collection = self.data_factory.create_collection_sample().model
+        item = self.data_factory.create_item_sample(collection=collection, name='base-bbox').model
+        asset = self.add_asset(item, 'har', None, None, None)
+        asset.geoadmin_variant = "krel"
+        asset.full_clean()
+        asset.save()
```

Comment:
Finally an assertion at the end that check the collection summaries with the "krel" variant would be good

Reply:
ha - I forgot assertions :-) But it works without 500er ...